spec/controllers/hyrax/admin/permission_template_accesses_controller_spec.rb

```diff
@@ -4,6 +4,7 @@
   let(:hyrax) { Hyrax::Engine.routes.url_helpers }
   let(:permission_template_access) { FactoryBot.create(:permission_template_access) }
   let(:source_id) { permission_template_access.permission_template.source_id }
+  let(:user_liz) { FactoryBot.create(:user, email: 'liz@example.com') }
```

Comment:
Without a FactoryBot created user, some of the actions did not have their methods executed because they failed `load_and_authorize_resource` in the class.